extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -636,10 +639,16 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
         .then(results => {
           const rtcParams = this.mergeRtcResponses_(results[0]);
           this.identityToken = results[1];
+          const pageParams = Object.assign(
+              PAGE_LEVEL_PARAMS_,
+              {
+                'gct': this.getLocationQueryParameterValue('google_preview') ||
+                    null,
+              });
           return googleAdUrl(
               this, DOUBLECLICK_BASE_URL, startTime, Object.assign(
                   this.getBlockParameters_(), rtcParams,
-                  this.buildIdentityParams_(), PAGE_LEVEL_PARAMS_));
+                  this.buildIdentityParams_(), pageParams));
```

Comment:
does this not need to be done for SRA?

Reply:
Good catch!  I make a getPageParameters_ function to ensure consistency while allowing for easy testing.